src/dotnet/install.sh

```diff
@@ -105,7 +105,7 @@ done
 
 # Install .NET versions and dependencies
 # icu-devtools includes dependencies for .NET
-check_packages wget ca-certificates icu-devtools
+check_packages wget ca-certificates icu-devtools curl jq
```

Comment:
Perhaps you could drop `curl` and only use `wget` for consistency with existing code